### PR TITLE
highcharts-ng

### PR DESCRIPTION
**Pull request by scatcher**
Initial push of highcharts-ng definition and supporting tests.

### CODE REVIEW

File: highcharts-ng/highcharts-ng-tests.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+			},
+			credits: {
+				enabled: false
+			},
+			plotOptions: {}
+		},
+		series: [{
+            data: [10, 15, 12, 8, 7]
+        }],
+		title: {
```
Comment (DanielRosenwasser) [inline highcharts-ng/highcharts-ng-tests.ts]: This looks awkward, can you use spaces and normalize the indentation?

**Comment by scatcher** [comment]
It looks like the last update of VS Code decided that I'd like to start using tabs for spacing, should be fixed now.

**Comment by vvakame** [comment]
*highcharts-ng/highcharts-ng.d.ts*

check list

* [ ] is collect [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)?
  * https://www.npmjs.com/package/highcharts-ng
  * http://bower.io/search/?q=highcharts-ng
  * others?
* [X] has a [test file](http://definitelytyped.org/guides/contributing.html#tests)? (highcharts-ng/highcharts-ng-tests.ts or others)
* [ ] pass the Travis-CI test?

**Comment by vvakame** [comment]
@scatcher thanks mate!

**Pull request merged**